cdk/visit/lambda_code/log_visit/log_visit.py

```diff
@@ -1,5 +1,6 @@
 import json
 import datetime
+from xmlrpc import client
```

Comment:
This feels like its likely the wrong import. I would expect the client type we are referring to to live in the boto package.

Reply:
Removed the import since it was unused.